doc/pages/Manual.md

```diff
@@ -677,9 +677,6 @@ Additionally, the number of hexadecimal chars must match exactly what is
 expected by the corresponding algorithm (resp. 32, 64 and 128 for MD5, SHA256
 and SHA512).
 
-If `openssl` is installed on the system, it will be used for faster computation
```

Comment:
The documentation is global, better rephrasing with *Until opam 2.2.0, if openssl...*

Reply:
Fixed